draft-ietf-taps-impl.md

```diff
@@ -435,7 +435,7 @@ It is useful to process success and failure throughout the tree by child nodes r
 ...
 ~~~~~~~~~~
 
-If a leaf node has successfully completed its connection, all other attempts should be made ineligible for use by the application for the original request. New connection attempts that involve transmitting data on the network should not be started after another leaf node has completed successfully, as the connection as a whole has been established. An implementation may choose to let certain handshakes and negotiations complete in order to gather metrics to influence future connections. Similarly, an implementation may choose to hold onto fully established leaf nodes that were not the first to establish for use in future connections, but this approach is not recommended since those attempts were slower to connect and may exhibit less desirable properties.
+If a leaf node has successfully completed its connection, all other attempts should be made ineligible for use by the application for the original request. New connection attempts that involve transmitting data on the network should not be started after another leaf node has completed successfully, as the connection as a whole has been established. An implementation may choose to let certain handshakes and negotiations complete in order to gather metrics to influence future connections. Similarly, an implementation may choose to hold onto fully established leaf nodes that were not the first to establish for use as part of a Pooled Connection, see {{pooled-connections}}, or in future connections. The latter is not recommended since those attempts were slower to connect and may exhibit less desirable properties.
```

Comment:
I don't think that mentioning the pooled connections here is really of too much value—this sentence is about connections that lost races (using worse addresses or interfaces). Do we really want to recommend using these for pools?

Reply:
Depends on the purpose of the pool:
 – If it is to improve reliability, keeping a second worse connection as backup makes sense.
 - If it is to optimise on a per-request basis, there might be one low-latency low-throughput and one high-latency high-throughput interface. The latter will loose the race, but may be more desirable for some requests.
I will update the text up make this clearer.

Reply:
The new text has a couple typos (see other comment), but I still don't totally get the example scenarios described in which this would be wanted. How about just saying implementations may use them, but they're not recommended in general?

Reply:
fine with me